saas/models/saas_operator.py

```diff
@@ -52,3 +54,33 @@ def get_db_name(self, db):
         # TODO: use mako for url templating
         self.ensure_one()
         return self.db_url_template.format(db_id=db.id)
+
+    def get_auth_urls(self, db):
+        self.ensure_one()
+        if self.type == 'local':
+            master_url = self.env['ir.config_parameter'].get_param('web.base.url')
+            build_url = self.get_db_url(db)
+        return master_url, build_url
+
+    def build_execute_kw(self, build, code):
+        if self.type == 'local':
+            db = sql_db.db_connect(build.name)
+            registry(build.name).check_signaling()
+            with api.Environment.manage(), db.cursor() as cr:
+                env = api.Environment(cr, SUPERUSER_ID, {})
+                action = env['ir.actions.server'].create({
+                    'name': 'Local Code Eval',
+                    'state': 'code',
+                    'model_id': 1,
+                    'code': code,
+                })
+                action.run()
+
+    @job
+    def build_post_init(self, build, post_init):
+        master_url, build_url = self.get_auth_urls(build)
+        mandatory_post_init = 'action = env[\'ir.config_parameter\'].create([{\'key\': \'auth_quick.master\', \'value\': \''\
+                              + master_url + '\'}, {\'key\': \'auth_quick.build\', \'value\': \'' + build_url + '\'}])'
+        post_init.append(mandatory_post_init)
```

Comment:
You might need to to insert values of ``build_post_init_ids`` here. Probably via ``.format(**key_value_dict)``.
For example, user can set hist post_init as following 

```
# set company name
self.env.user.company_id.name = "{COMPANY_NAME}"
```